lbry/wallet/server/db/writer.py

```diff
@@ -55,6 +55,7 @@ class SQLDB:
             description text,
 
             claim_type integer,
+            no_source bool,
```

Comment:
instead of adding a `no_source` field, would it be more flexible to add a `source` field with the actual source, that would have the added benefit of being able to search for claims with particular source?

Reply:
`source` is a dictionary with [more than one field](https://github.com/lbryio/types/blob/master/v2/proto/claim.proto#L56) right? unless you have one of them in mind.

I like the idea but it would be nice to do this in a way where we don't need one column for each subitem. (something like a [flattened field](https://www.elastic.co/guide/en/elasticsearch/reference/master/flattened.html) maybe)